docs/CHANGELOG.rst

```diff
@@ -27,6 +27,8 @@ Fixed
 -----
 - Enable ordering on knowledge-base endpoints
 - Fix the check for genome builds of inputs in process ``snpeff``
+- Fix per-lane rc output in ``feature_counts-beta`` and featureCounts
```

Comment:
Since this is a fix for yet unreleased feature (mentioned under "Added" section), an additional chengelog entry is not necessary.